os_pkrvars/windows/windows-11-x86_64.pkrvars.hcl

```diff
@@ -3,8 +3,8 @@ os_version = "11"
 os_arch    = "x86_64"
 is_windows = true
 # Windows 11 source requires a prompt to select OS to install. This allows the system time to add reg keys to ignore TPM Check before the install begins.
-iso_url                 = "https://software-static.download.prss.microsoft.com/dbazure/988969d5-f34g-4e03-ac9d-1f9786c66751/22621.525.220925-0207.ni_release_svc_refresh_CLIENTENTERPRISEEVAL_OEMRET_x64FRE_en-us.iso"
-iso_checksum            = "ebbc79106715f44f5020f77bd90721b17c5a877cbc15a3535b99155493a1bb3f"
+iso_url                 = "https://software-static.download.prss.microsoft.com/dbazure/888969d5-f34g-4e03-ac9d-1f9786c66749/22631.2428.231001-0608.23H2_NI_RELEASE_SVC_REFRESH_CLIENTENTERPRI>
```

Comment:
looks like URL is trucated.

Reply:
Yes, I copy&pasted from nano (facepalm). The fixup was already there :)